Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md

```diff
@@ -0,0 +1,99 @@
+# Applying Dashboard Parameters for Widget Embedding using .NET Core
+
+This project was created using ASP.NET Core 6.0. This application aims to demonstrate how to showcase the implementation of dashboard parameters in a BoldBI widget through JavaScript Embedding.
+
+## Dashboard view
+
+![Dashboard View](https://github.com/boldbi/aspnet-core-sample/assets/91586758/6c03cba1-beeb-4be2-8a9e-f77f6a3ebf91)
+
+ ## Requirements/Prerequisites
+
+ * [.NET Core 6.0](https://dotnet.microsoft.com/download/dotnet-core)
+ * [Visual Studio Code](https://code.visualstudio.com/download)
+ 
+ #### Supported browsers
+  
+  * Google Chrome, Microsoft Edge and Mozilla Firefox.
+
+ ## Configuration
+
+  * Please ensure you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.
+
+    ![Embed Settings](https://github.com/boldbi/aspnet-core-sample/assets/91586758/b3a81978-9eb4-42b2-92bb-d1e2735ab007)
+
+  * To download the `embedConfig.json` file, please follow this [link](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) for reference. Additionally, you can refer to the following image for visual guidance.
+
+     ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
```

Comment:
Use newly created sample URL as I reported previously.

Reply:
Changed the URL and resolved the concerns